sky/clouds/service_catalog/images/provisioners/cuda-azure-grid.sh

```diff
@@ -11,6 +11,11 @@ wget -nv $GRID_DRIVER_URL -O $GRID_DRIVER_FILE
 sudo chmod +x $GRID_DRIVER_FILE
 sudo sh $GRID_DRIVER_FILE --silent --disable-nouveau
 
+echo "Set vGPU Licensing Daemon config..."
+sudo cp /etc/nvidia/gridd.conf.template /etc/nvidia/gridd.conf
+sudo sed -i '/^FeatureType=0/s/^/# /' /etc/nvidia/gridd.conf
+echo "IgnoreSP=FALSE" | sudo tee -a /etc/nvidia/gridd.conf
+echo "EnableUI=FALSE" | sudo tee -a /etc/nvidia/gridd.conf
```

Comment:
I am not sure why, but when I create an instance with the `sky launch -c test-azure-a10 --cloud azure --gpus A10 --image-id /CommunityGalleries/skypilot-5c783dcb-c1d8-48ca-a5eb-0cee59e165fd/Images/skypilot-gpu-gen2-grid`. The `/etc/nvidia/gridd.conf` file only has the following. Is that expected?
```
EnableUI=FALSE
```

Reply:
I regenerated the image so things should look right now